### PR TITLE
Menu, Actions and Event capitalization standardization

### PR DESCRIPTION
**Pull request by jwrober**
No related issue. Took opportunity to clean up the capitalization and also some menu item names. No functional change, just cosmetic.

### CODE REVIEW

File: client/menu.cpp [hunk 1]
```diff
@@ -258,7 +258,7 @@ void gov_menu::update_all()
 /**
    Instantiate a new goto and act sub menu.
  */
-go_act_menu::go_act_menu(QWidget *parent) : QMenu(_("Go to and..."), parent)
+go_act_menu::go_act_menu(QWidget *parent) : QMenu(_("Goto And..."), parent)
```
Comment (lmoureaux) [inline client/menu.cpp:261]: Should we really merge `goto` or is it informal usage? My spellchecker isn't happy.

**Review by lmoureaux — COMMENTED** [review]
Do we need to update the game manual to match?

Also one specific comment (appearing twice).

**Comment by jwrober** [comment]
> Do we need to update the game manual to match?

Yea. I am working on another commit for the manual.

File: client/menu.cpp (hunk 1 above)
Comment (jwrober) [reply client/menu.cpp:261]: I did a quick search, seems "go to" is the correct way. I'll fix.

**Pull request merged**